module_1/task2/index.js

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import csv from 'csvtojson';
+import fs from 'fs/promises';
+
+const buildFilePath = filePath => path.join(__dirname, filePath);
+
+const csvFilePath = 'csv/test.csv';
+const csvToTxtFilePath = 'cvsToTxt.txt';
+
+const fullCsvFilePath = buildFilePath(csvFilePath);
+const fullCsvToTxtFilePath = buildFilePath(csvToTxtFilePath);
```

Comment:
both are calculated dynamic values that might be inside main function. e.g.:

```javascript
async function checkAndParseTxtFileToCsv(pathToFile) {
  const fullCsvToTxtFilePath = buildFilePath(pathToFile);
  
  try {
    const csvFileStat = await fs.stat(fullCsvToTxtFilePath);

    if (csvFileStat.isFile()) {
      await fs.unlink(fullCsvToTxtFilePath);
    }
  } catch (e) {
    if (e.code !== 'ENOENT') {
      console.log(e);
      return;
    }
  }

  parseCsv();
}
```

Reply:
similarly for `parseCsv`...

Reply:
```
const csvFileStat = await fs.stat(fullCsvToTxtFilePath);

    if (csvFileStat.isFile()) {
      await fs.unlink(fullCsvToTxtFilePath);
    }
```
can be moved to a separate helper - `unlinkFileIfExist`...

Reply:
thanks, added commit with updates